MapboxCoreNavigation/MGLMapboxEvents.swift

```diff
@@ -0,0 +1,39 @@
+import Mapbox
+import Polyline
+
+extension MGLMapboxEvents {
+    class func addDefaultEvents(routeProgress: RouteProgress, sessionIdentifier: UUID) -> [String: Any] {
+        var modifiedEventDictionary: [String: Any] = [:]
+    
+        modifiedEventDictionary["platform"] = UIDevice.current.systemName
```

Comment:
This library is only for iOS, so `#if os(iOS)` is unnecessary. The issue is actually that Core Navigation avoids depending on UIKit. We should add a hook for MapboxNavigation to add to this dictionary.